### ISSUE TITLE
Generate KBART records

### ISSUE THREAD

**Issue by vincentwj** [thoth-pub/thoth#88]
Recommendation 49. COPIM must review the resources made available on KBART and seek to seek endorsement of its files.

Recommendation 50. Once KBART files are available, COPIM should test with partner institutions that it is available as a package to ‘subscribe’ to in library discovery systems.

**Comment by vincentwj** [comment]
I think there is difference between our KBART format compying to NISO standards and the records of a publisher in that format being compliant with it?

**Comment by rhigman** [comment]
The KBART (NISO) Standing Committee have now also sent feedback, some of which I was able to implement/had already implemented based on other feedback:
- They echoed Jisc in requesting that only one author/editor be listed in `first_author`/`first_editor`, and reiterated that `date_monograph_published_online` and `date_monograph_published_print` are mandatory (already fixed since sending the example)
- They requested that data cells not be enclosed in quotation marks (fixed under https://github.com/thoth-pub/thoth/pull/277/commits/a4d92242b1b609a668c41456e55df633e82d2a08)
- They pointed out that the filename does not match the NISO guidelines, as above - we will need to resolve this if we eventually set up automated distribution

Some of their feedback was specific to the example data I had sent, which was user-submitted and not always fully accurate/appropriate. I therefore explained the relation between Thoth and individual publishers, and asked whether this would affect the way the endorsement process worked (cf Vincent's comment above).
- Notably, they requested we don't use DOIs for `title_id`, but instead use the product identifier numbers from title URLs (e.g. `137` for `https://www.openbookpublishers.com/product/137`), as the example file I had sent was based on OBP data. I explained that not all Thoth publishers would have the same title URL structure, and asked whether DOIs would be acceptable on this basis.
